docs/StringManifesto.md

```diff
@@ -982,7 +982,7 @@ development.
 
 #### Printf-Style Formatting
 
-`String.format` is designed on the `printf` model: it takes a format string with
+`String(format:)` is designed on the `printf` model: it takes a format string with
```

Comment:
I think this change is correct, but I'm not 100% sure. I suppose by `String.format` you meant `String.init(format:)`, right?